client/src/core/ErrorBoundary.js

```diff
@@ -61,10 +63,12 @@ export class ErrorBoundary extends React.Component {
     throw this.state.error;
   }
   render() {
+    const { children } = this.props;
     const { error, testing_for_stale_client } = this.state;
+    const redirected_msg = sessionStorage.getItem("redirected_msg");
```

Comment:
Still using `sessionStorage` rather than app store? Here and two other places. Are those leftovers, or was some mix necessary in the end?